vsts/pipelines/ci.yml

```diff
@@ -1,3 +1,32 @@
+parameters:
+  - name: buildImages
+    type: object
+    default:
+      - 
+        key: Latest
+        value: latest
+      - 
+        key: LtsVersions
+        value: ltsversions
+      - 
+        key: Jamstack
+        value: jamstack
+      - 
+        key: GithubActions
+        value: githubactions
+      - 
+        key: VsoFocal
+        value: vso-focal
```

Comment:
Side note: 
JobNames cannot have a dash `-`, [docker images cannot have uppercases](https://github.com/docker/compose/issues/1416#:~:text=We%20allow%20uppercase%20letters%20in,will%20refuse%20to%20create%20it.), and having underscore brings problems with hostnames. Which is why we have the key/value parameters as shown above.